Solutions/Marain.UserNotifications.Client/Marain/UserNotifications/Client/Management/Resources/CommunicationTemplates/WebPushTemplateResource.cs

```diff
@@ -40,6 +40,23 @@ public WebPushTemplateResource(in HalDocumentResource source)
         /// <inheritdoc/>
         public WebLink SelfLink => this.HalDocument.SelfLink;
 
+        /// <summary>
+        /// Gets the notification type of a WebPush notification.
+        /// </summary>
+        public string NotificationType
+        {
+            get
+            {
+                if (this.HalDocument.TryGetProperty("notificationType", out JsonElement notificationType) &&
+                    notificationType.ValueKind == JsonValueKind.String)
+                {
+                    return notificationType.GetString();
+                }
+
+                throw new Exception("Schema violation - this value is required.");
```

Comment:
Same comment as earlier.

Reply:
Replied above, lets rather track the issue in the other comment